plugins/source/aws/docs/index.md

```diff
@@ -1,10 +1,10 @@
-# AWS Provider
+# AWS Plugin
 
 The CloudQuery AWS provider extracts and transforms your AWS cloud assets configuration into PostgreSQL.
 
 ## Install
 
-```shell
```

Comment:
Renamed to `bash` as `shell` is not supported by Nextra: https://github.com/FormidableLabs/prism-react-renderer/blob/master/src/vendor/prism/includeLangs.js